Project/Source Files/Screens/Meal/MealCollectionViewController.swift

```diff
@@ -35,6 +35,7 @@ final class MealCollectionViewController: UIViewController, UICollectionViewData
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        customView.setBlurLoader()
```

Comment:
Let's put this line just above the API call so it will reflect that it is tightly connected to this functionality